interfaces/builtin/broadcom_asic_control.go

```diff
@@ -38,12 +38,20 @@ const broadcomAsicControlConnectedPlugAppArmor = `
 /dev/linux-user-bde rw,
 /dev/linux-kernel-bde rw,
 /dev/linux-bcm-knet rw,
+
+# These are broader than they needs to be, but until we query udev
+# for specific devices, use a broader glob
```

Comment:
@niemeyer, @adglkh - they are necessarily broad because in PR review it was found there was nothing in the /sys/devices path alone that made it 'broadcom-asic'. I believe it is technically possible for snapd to interrogate the system via udev to get the precise /sys/devices and /run/udev/data paths for the device (see https://github.com/snapcore/snapd/pull/1226#r66140667 for back of the napkin idea), but this work is neither prioritized nor assigned. As a result, this interface along with existing opengl and camera interfaces use rules that are broader than they should be (but we comment in the policy in camera and opengl about the leak). While the accesses in question do constitute an information leak, they are not terribly significant (which is why the work is not prioritized/assigned) and in the 'future work' bucket.

I think the plan should be:
- continue with the access as written since without it, things break
- add a comment that this is an information leak (see opengl.go)
- at some future time, implement the udev querying in snapd and have new interfaces in series 16 use this
- in some future iteration of snappy which is generating different policy (eg, something not 'series 16'), migrate existing interfaces that use the broad globs to using only the outputs of udev querying